src/lib.rs

```diff
@@ -55,12 +55,20 @@ mod event_grab_broken;
 mod event_key;
 mod event_motion;
 mod event_owner_change;
+#[cfg(any(feature = "v3_22", feature = "dox"))]
+mod event_pad_axis;
+#[cfg(any(feature = "v3_22", feature = "dox"))]
+mod event_pad_button;
+#[cfg(any(feature = "v3_22", feature = "dox"))]
+mod event_pad_group_mode;
 mod event_property;
 mod event_proximity;
 mod event_scroll;
 mod event_selection;
 mod event_setting;
 mod event_touch;
```

Comment:
`GDK_TOUCH_BEGIN/END/CANCEL/UPDATE` exist since 3.4, not sure how this worked without feature flag before?